### PR TITLE
Allow conda activate to set its own PATH.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: scripts/newinstall.sh [hunk 1]
```diff
@@ -919,10 +919,9 @@ n8l::generate_loader_bash() {
 	if [[ -n $miniconda_path ]]; then
 		local cmd_setup_miniconda
 		cmd_setup_miniconda="$(cat <<-EOF
-			export PATH="${miniconda_path}/bin:\${PATH}"
 			export LSST_CONDA_ENV_NAME=\${LSST_CONDA_ENV_NAME:-${LSST_CONDA_ENV_NAME}}
 			# shellcheck disable=SC1091
-			source activate "\$LSST_CONDA_ENV_NAME"
+			source "${miniconda_path}/bin/activate" "\$LSST_CONDA_ENV_NAME"
```
Comment (timj) [inline scripts/newinstall.sh]: At some point aren't we supposed to be switching to `conda activate` ?

**Comment by ktlim** [comment]
@jhoblitt Indeed the point of this is to change the template output.  I believe I have removed the test that no longer applies.

File: scripts/newinstall.sh (hunk 1 above)
Comment (ktlim) [reply scripts/newinstall.sh]: I'll leave that modernization to someone else.

**Comment by ktlim** [comment]
@jhoblitt Some help with figuring out why the bash compile fails would be appreciated, though.  The failure seems to be caused by the makefile looking for `newversion.h` after it has explicitly moved it aside.  Perhaps a parallel make race condition?  Nothing seems new for this step.

**Comment by ktlim** [comment]
Looks like making the `make` non-parallel solved this.

**Pull request merged**